pallets/hasher/src/benchmarking.rs

```diff
@@ -29,7 +29,7 @@ use frame_support::traits::Currency;
 use frame_system::RawOrigin;
 use sp_runtime::traits::Bounded;
 use webb_primitives::types::DepositDetails;
-fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::Event) {
+fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::RuntimeEvent) {
```

Comment:
We have this function written 5 or so times, in the future @salman01zp please break it out and de-duplicate the code. Similar to what we are doing with webb.js/test-utils we should be doing something similar here. Creating a test-utils package that eventually lives and gets used by various substrate/rust projects (here it would be Substrate specific for example).